QBImagePicker/QBAlbumsViewController.m

```diff
@@ -92,10 +91,12 @@ - (void)resetColours {
 	if (@available(iOS 13.0, *)) {
 
 		if ([UITraitCollection currentTraitCollection].userInterfaceStyle == UIUserInterfaceStyleDark) {
+			[self.doneButton setTitleTextAttributes:@{NSForegroundColorAttributeName : kDarkBackground} forState:UIControlStateDisabled];
```

Comment:
How will it behave on iOS 12 ? I ask because all of this code is wrapped into a `if (@available(iOS 13.0, *))`

Reply:
It would behave as before, the code within the `else` is basically the original code before supporting darkmode

Reply:
Nevermind... I just realised my mistake